modules/install/pages/upgrade-strategy-for-features.adoc

```diff
@@ -0,0 +1,71 @@
+= Upgrading to Couchbase Server 6.0
+
+Upgrading to the latest version of Couchbase Server allows you to leverage the latest features and helps keep your cluster running well.
+The following table summarizes the new features in this release, and describes what parts of the cluster need to be upgraded to enable them.
+
+.Couchbase Server 6.0 Feature Upgrade Matrix
+[cols="5,2,2,2,2,2,2"]
+|===
+| Feature | Upgrade Level | xref:upgrade-strategies.adoc[Upgrade Type] | xref:upgrade.adoc#table_swd_vpc_rbb[Upgrade Path] | Mixed Mode† | Dependencies | Additional Information
+
+| Analytics Service
+| Cluster
+| Offline/Online
+| Direct
+| No
+| If you plan to use the production release of Couchbase Analytics in version 6.0, you must perform a fresh installation of Couchbase Server 6.0 on any existing Analytics nodes that are running a previous version. Otherwise, the Analytics Service will not function properly.
+| Analytics data from Developer Preview releases cannot be upgraded.
+
+| Timers (Eventing Service)
+| Cluster
+| Offline/Online
+| Direct
+| No
+| All Eventing nodes must be running version 6.0 to enable this feature.
+| None
+
+| Scorch indexing system (Full Text Search)
+| Cluster
+| Offline/Online
+| Direct
+| Yes*
+| *FTS nodes running 6.0 use the Scorch indexing system by default for new indexes.
+You can continue to use previous indexes in 6.0, but they will not use Scorch.
+| To upgrade previous indexes to use Scorch, simply recreate them using the new index type option.
+|===
+
+[#feature-mixed-mode]
+† _Mixed Mode in this context refers to only the nodes that are running the indicated service. So if 'No' is marked in the Mixed Mode column of a given feature, then all nodes that are running that service must be upgraded to the same version in order to enable that feature._
+
+[#upgrade-ipv6]
+== Migrating Data from an IPv4 Cluster to an IPv6 Cluster
```

Comment:
Version number update from 5.5 to 6.0?

Reply:
I assume this is regarding the 5.5 references for IPv6. It's not specific to 6.0, but right now there isn't a good home for this information, so I left it in the same place it was for 5.5. Also there is a good chance these procedures will change soon.